presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -171,6 +197,17 @@ public void testRoundTripAllFormats(RoundTripTestCase testCase)
                 "VALUES (" + testCase.getFieldValues() + "), (" + testCase.getFieldValues() + ")");
     }
 
+    @Test
+    public void testKafkaHeaders()
```

Comment:
Move this before `testRoundTripAllFormats` so that it doesn't split that test from its data provider